### DISCUSSION TITLE
I'm here to manage my repositories. Why is Github trying to be Facebook?

### DISCUSSION THREAD

**Discussion by Bigfootmech** [community/community#64935] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Why the feed?
Why the unnecessary 2FA that doesn't even consider using SSH keys?

Who took Github out back and shot it?

**Comment by ghost** [comment]
Github becoming more and more frustrating every day. Looks like everything micro$oft touches becoming a piece of crap. Skype, Github, what is next ? Bethesda ?

**Comment by helloukeje** [comment]
I don't think I have any problem with it

**Comment by lee-b** [comment]
Microsoft bought it.

**Comment by zmic** [comment]
Github's 2FA is an incomprehensible mess and the only thing it will ever end up doing is lock me out from my account.
